### PR TITLE
HIA-426 - Refactoring models structure and import aliases

### PR DESCRIPTION
**Pull request by chiaramapellimt**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt [hunk 1]
```diff
@@ -6,23 +6,23 @@ import org.springframework.http.HttpMethod
...
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Sentence
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.SentenceAdjustment
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.SentenceKeyDates
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.UpstreamApi
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisBooking
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisImageDetail
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisOffenceHistoryDetail
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisOffenderSentence
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisSentenceSummary
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisAddress as AddressFromNomis
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt:23]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt [hunk 2]
```diff
@@ -6,23 +6,23 @@ import org.springframework.http.HttpMethod
...
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisAlert as AlertFromNomis
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt:24]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt [hunk 3]
```diff
@@ -6,23 +6,23 @@ import org.springframework.http.HttpMethod
...
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisSentence as SentenceFromNomis
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/NomisGateway.kt:25]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnNeed.kt [hunk 4]
```diff
@@ -1,8 +1,8 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Need as IntegrationApiNeed
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Need as IntegrationApiNeed
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnNeed.kt:3]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnNeeds.kt [hunk 5]
```diff
@@ -1,13 +1,13 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
 import java.time.LocalDateTime
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Needs as IntegrationApiNeeds
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Needs as IntegrationApiNeeds
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnNeeds.kt:4]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnOtherRisks.kt [hunk 6]
```diff
@@ -1,8 +1,8 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.OtherRisks as IntegrationApiOtherRisks
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.OtherRisks as IntegrationApiOtherRisks
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnOtherRisks.kt:3]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRiskSummary.kt [hunk 7]
```diff
@@ -1,8 +1,8 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.RiskSummary as IntegrationApiRiskSummary
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.RiskSummary as IntegrationApiRiskSummary
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRiskSummary.kt:3]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRiskToSelf.kt [hunk 8]
```diff
@@ -1,13 +1,13 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.RiskToSelf as IntegrationApiRiskToSelf
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.RiskToSelf as IntegrationApiRiskToSelf
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRiskToSelf.kt:3]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRisks.kt [hunk 9]
```diff
@@ -1,13 +1,13 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.assessRisksAndNeeds
 
 import java.time.LocalDateTime
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Risks as IntegrationApiRisks
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Risks as IntegrationApiRisks
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRisks.kt:4]: Is this named import still needed?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/assessrisksandneeds/GetNeedsForPersonTest.kt [hunk 10]
```diff
@@ -16,10 +16,10 @@ import uk.gov.justice.digital.hmpps.hmppsintegrationapi.gateways.AssessRisksAndN
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.gateways.HmppsAuthGateway
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.mockservers.AssessRisksAndNeedsApiMockServer
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.mockservers.HmppsAuthMockServer
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApiError
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.UpstreamApiError
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/assessrisksandneeds/GetNeedsForPersonTest.kt:19]: Is this named import still needed?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/PersonSentencesTest.kt [hunk 11]
```diff
@@ -2,13 +2,11 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
...
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.SentenceLength
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApi
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.SentenceLength
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.UpstreamApi
 import java.time.LocalDate
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Sentence as IntegrationApiSentence
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.SentenceTerm as IntegrationApiTerm
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.Sentence as NomisSentence
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.Term as NomisTerm
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Sentence as IntegrationApiSentence
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/PersonSentencesTest.kt:8]: Is this named import still needed?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceAdjustmentTest.kt [hunk 12]
```diff
@@ -2,7 +2,7 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
 
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.SentenceAdjustment as SentenceAdjustmentFromNomis
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisSentenceAdjustment as SentenceAdjustmentFromNomis
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceAdjustmentTest.kt:5]: Is this named import still needed?

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceKeyDatesTest.kt [hunk 13]
```diff
@@ -3,7 +3,7 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
 import java.time.LocalDate
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.SentenceKeyDates as SentenceKeyDatesFromNomis
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisSentenceKeyDates as SentenceKeyDatesFromNomis
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceKeyDatesTest.kt:6]: Is this named import still needed?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/NomisOffender.kt [hunk 14]
```diff
@@ -1,11 +1,11 @@
 package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis
 
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Alias
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Person
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Alias
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Person
 import java.time.LocalDate
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.Alias as nomisAlias
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisAlias as nomisAlias
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/NomisOffender.kt:6]: One more named import

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/AlertTest.kt [hunk 15]
```diff
@@ -2,8 +2,9 @@ package uk.gov.justice.digital.hmpps.hmppsintegrationapi.models
 
 import io.kotest.core.spec.style.DescribeSpec
 import io.kotest.matchers.shouldBe
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Alert
 import java.time.LocalDate
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.Alert as AlertFromNomis
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.nomis.NomisAlert as AlertFromNomis
```
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/AlertTest.kt:7]: One more

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ProbationIntegrationEPFGateway.kt [hunk 16]
```diff
@@ -6,10 +6,10 @@ import org.springframework.http.HttpMethod
 import org.springframework.stereotype.Component
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.extensions.WebClientWrapper
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.extensions.WebClientWrapper.WebClientWrapperResponse
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.Response
-import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.UpstreamApi
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.Response
+import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.hmpps.UpstreamApi
 import uk.gov.justice.digital.hmpps.hmppsintegrationapi.models.probationintegrationepf.CaseDetail
```
Comment (CslBing) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ProbationIntegrationEPFGateway.kt:11]: Can this upstream model renamed?

Comment (CslBing) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ProbationIntegrationEPFGateway.kt:11]: And remove  the named import the line below

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceAdjustmentTest.kt (hunk 12 above)
Comment (CslBing) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/nomis/SentenceAdjustmentTest.kt:5]: This one still here?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ProbationIntegrationEPFGateway.kt (hunk 16 above)
Comment (chiaramapellimt) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/gateways/ProbationIntegrationEPFGateway.kt:11]: yep will have a think

**Review by CslBing — APPROVED** [review]

**Pull request merged**